modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Link.es.js

```diff
@@ -9,5 +9,9 @@ import {Link as RouterLink} from 'react-router-dom';
 import {stringToSlug} from '../utils/utils.es';
 
 export default function Link(props) {
+	if (!props.slugTo) {
```

Comment:
We can use a ternary condition here, like this is more clear I think:  
`return props.slugTo ? ( <RouterLink {...props} to={stringToSlug(props.to)} /> ) : ( <RouterLink {...props} to={props.to} /> );`